src/governance.ts

```diff
@@ -128,8 +134,54 @@ export class GovernanceCanister {
     return this.responseConverters.toListProposalsResponse(rawResponse);
   };
 
+  public createNeuron = async ({
```

Comment:
```suggestion
  public stakeNeuron = async ({
```

nit: this is more consistent with our terminology.

Reply:
ok!